shell.nix

```diff
@@ -33,14 +33,15 @@ mkShell {
     jq
     llvmPackages.libclang
     nixpkgs-fmt
-    openapi-generator
+    openapi-generator-cli
```

Comment:
Since we are not going to use `openapi-generator-cli`, better remove from here?

Reply:
ok, from another comment I understand it'll still be used for python bdd tests.

Reply:
Yes, still need it, but now it's from upstream and cached by hydra